SpatialGDK/Source/Public/SpatialConstants.h

```diff
@@ -53,4 +61,6 @@ namespace SpatialConstants
 
 	const FString LOCAL_HOST = TEXT("127.0.0.1");
 	const uint16 DEFAULT_PORT = 7777;
+
+	const float ENTITY_QUERY_RETRY_WAIT_SECONDS = 3.f;
```

Comment:
```suggestion
	const float ENTITY_QUERY_RETRY_WAIT_SECONDS = 3.0f;
```

Reply:
Literally asked what we prefer just before I did this!